pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -270,6 +272,16 @@ async def run(self):
             except CalledProcessError as ex:
                 _LOGGER.warning(f"Unable to move {impetus} advisory {advisory} to QE: {ex}")
 
+        _LOGGER.info("Check failed greenwave tests...")
+        self._slack_client.bind_channel(self.release_name)
+        for advisory in [advisories["image"], advisories["extras"], advisories["metadata"]]:
```

Comment:
This will cause an exception if one of the hardcoded impetus doesn't exist. What will happen if checking for all advisories with a loop?